models/FollowModel.js

```diff
@@ -17,8 +17,37 @@ const getFollowStatus = `
   ) as "followStatus";
 `;
 
+const getFollowersUserList = `
+  SELECT 
+    u."userId", 
+    u."username", 
+    u."displayName", 
+    CASE
+      WHEN f2."followerUserId" IS NOT NULL THEN TRUE
+      ELSE FALSE
+    END AS "isFollowing"
+  FROM app_user u
+  INNER JOIN follow f1 ON u."userId" = f1."followerUserId"
+  LEFT JOIN follow f2 ON u."userId" = f2."followedUserId" AND f2."followerUserId" = $2
+  WHERE f1."followedUserId" = $1;
+`;
+
+const getFollowingUserList = `
+  SELECT
+    u."userId",
+    u."username",
+    u."displayName",
+    CASE WHEN f2."followerUserId" IS NOT NULL THEN TRUE ELSE FALSE END AS "isFollowing"
+  FROM app_user u
+  INNER JOIN follow f ON u."userId" = f."followedUserId"
```

Comment:
Nit, can we follow the same naming pattern as `getFollowersUserList`? Like either use `f1` or `f` for both

Reply:
Done